src/TreeHouse/Queue/Amqp/Driver/Amqp/Publisher/AmqpMessagePublisher.php

```diff
@@ -59,7 +59,11 @@ public function publish(Message $message, \DateTime $date = null, $flags = Excha
         $body = $message->getBody();
         $route = $message->getRoutingKey();
         $props = $message->getProperties()->toArray();
-
+        
+        if (!$this->exchange->getConnection()->isConnected()) {
+            $this->exchange->getConnection()->reconnect();
+        }
```

Comment:
does this really help? if I look into the called method it just returns the status code of the last command that was sent, which doesn't help if the connection is closed after that